### PR TITLE
remove styled

### PR DESCRIPTION
**Pull request by renancaraujo**

### CODE REVIEW

File: src/templates/flameconTemplate.js [hunk 1]
```diff
@@ -11,282 +9,119 @@ import {
...
-    minute: "numeric",
+  const [localTime, setLocalTime] = useState(null)
+
+  useEffect(() => {
+    const theDate = new Date(Date.UTC(2020, 8, 16, 20, 0))
+    const formatter = new Intl.DateTimeFormat(navigator.language, {
+      hour: "numeric",
+      minute: "numeric",
+    })
+    console.log("ah")
```
Comment (luanpotter) [inline src/templates/flameconTemplate.js:23]: ah

**Review by luanpotter — APPROVED** [review]
lgtm with one nit

File: src/templates/flameconTemplate.js (hunk 1 above)
Comment (luanpotter) [inline src/templates/flameconTemplate.js:23]:
```suggestion
```

File: src/templates/flameconTemplate.js [hunk 2]
```diff
@@ -20,7 +20,6 @@ export default function Template() {
       hour: "numeric",
       minute: "numeric",
     })
-    console.log("ah")
```
Comment (erickzanardo) [inline src/templates/flameconTemplate.js]: Nice! haha

**Pull request merged**